lib/node/spvnode.js

```diff
@@ -220,64 +203,11 @@ class SPVNode extends Node {
    * Scan for any missed transactions.
    * Note that this will replay the blockchain sync.
    * @param {Number|Hash} start - Start block.
-   * @param {Bloom} filter
-   * @param {Function} iter - Iterator.
-   * @returns {Promise}
-   */
-
```

Comment:
Would the removal of this code prevent any wallet rescans?

Reply:
SPV never really rescanned since it doesn't have block data. All it can do is reset and redownload the chain. The reset logic remains and will be run in wallet as a plug-in mode. The only thing actually killed by this PR is if the wallet is run as a separate process against an spv node (which is problematic for other reasons) and requests a rescan.

Reply:
Thank you for clarifying @pinheadmz, makes sense.